sources/packages/web/src/components/generic/StatusInvoiceBatchApproval.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <chip-status :status="chipStatus" :label="status" />
+</template>
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue";
+import ChipStatus from "@/components/generic/ChipStatus.vue";
```

Comment:
Minor. We can add this `StatusChip` to main.ts as it is very generic.

Reply:
Added to the main and changed existing files.

Reply:
Thank you very much.